contents/blog/product-led-growth.md

```diff
@@ -0,0 +1,193 @@
+
+## The Key is in the DNA
+
+While this "strategy" sounds more like common sense than a methodical approach, there's more to Product-led growth than meets the eye.
+
+To truly succeed as a business that drives growth primarily through its product, this concept needs to be built into the company culture.
+
+While Sales and Marketing can often work well as segregated teams that draw from the product but operate independently in practice, focusing on building a product is an organization-wide effort.
+
+As PLGC put it, different teams often operate in different wavelengths, but with Product-Led Growth, you need everyone to converge into one single wavelength: user experience. 
+
+At PostHog, we follow a Product-led Growth approach. And we'll tell you, this has implications for all areas of the company:
+
+### Sales
+
+Don't make a sale if your product is not a good fit for the customer. 
+
+From the first paragraph of PostHog's Handbook Sales page:
+
+<blockquote class="blog-quote">
+
+Always focus on delivering what the customer needs. Sometimes that will mean sending them to a competitor or turning them down.
+
+</blockquote>
+
+Yes, this is literally an official guideline for doing Sales at PostHog.
+
+### Team
+If you're a software company, PLG might mean opting for more Engineers than Sales people, for instance. In our case, we still don't have a single person doing Sales or Marketing exclusively. Almost our entire team is made up of Engineers, including people not working day-to-day on our codebase. 
+
+More than with other approaches, you also need to make sure your hires are individuals who understand and are passionate about the product, as well as are proactive in making it better. 
+
+Your Sales people shouldn't cut corners to close a deal, and the Marketing team should focus on highlighting the product you built, not the brand you are. Your brand is the product.
+
+### Communication
+Listen to your users on everything.
+
+Built something? Get feedback. New idea? Get feedback. Launched a release? Get feedback.
+
+As one of our core devs put it following interviews with users about a new feature he built: "Your assumptions are mostly wrong. Talk to people to correct them".
+
+When generating growth primarily through Sales or Marketing, the segregation between the people who are building the product and those selling it can lead to diversion between what users want and what is being offered to them.
+
+When focusing on the product, you should always be touching base with your users, and be willing to drop features you love if they don't feel the same way.
+
+### Priorities
+
+At PostHog, our methodology is:
+
+<blockquote class="blog-quote">
+
+If we keep our team first and users second, then our investors will take care of themselves!
+
+</blockquote>
+
+Users are key and should be a top priority.
```

Comment:
The actual intention behind this statement is more:

We just need to find a kick ass group of special people and get them to work in the best place of their lives, they'll tell us what to do AND it'll be something we're all proud of.

versus 'we should dogfood', which we should